README.md

```diff
@@ -1,7 +1,7 @@
 # Martian Rust Adapter
 
 
-[![Build Status](https://img.shields.io/travis/martian-lang/martian-rust/master?logo=travis&style=for-the-badge)](https://travis-ci.org/martian-lang/martian-rust)
+[![Build Status](https://github.com/martian-lang/martian-rust/actions/workflows/rust.yml/badge.svg)](https://github.com/martian-lang/martian-rust/actions/workflows/rust.yml)
```

Comment:
Hm, is there a form of this that gives more consistent styling with the documentation link?

Reply:
You can't tweak the github svg AFAIK (no option the UI that gives you the link) but you can make the documentation badge match the github action link... so I did that.